index.html

```diff
@@ -1447,24 +1431,44 @@ <h2>The <dfn>NFCRecordType</dfn> enum</h2>
       Promise&lt;void&gt; push(NFCMessageSource message, optional NFCPushOptions options);
     };
 
-    [SecureContext]
-    interface NFC {
-      Promise&lt;void&gt; watch(MessageCallback callback, optional NFCWatchOptions options);
+    [Constructor(), SecureContext, Exposed=Window]
+    interface NFCReader : EventTarget {
+      Promise&lt;void&gt; watch(optional NFCWatchOptions options);
+      attribute EventHandler onreading;
```

Comment:
I'm not sure about the "reading" name; I will have to think about it more. If others like it, I'm fine with it.

Reply:
I see, "reading" is used all over the generic sensor API--makes sense here.

Reply:
Yup that is why I choose it